### DISCUSSION TITLE
Add Wireframes for 2D Mode

### DISCUSSION THREAD

**Discussion by yosimba2000** [godotengine/godot-proposals#10204] [2D]
I'm working in the 2D mode, and have a subdivided plane. Unfortunately, I cannot see the actual subdivisions since wireframes are unavailable.

Please make wireframes available in 2D.

My use case: I'm making a shader that displaces vertices on the mesh, and I need to visibly see the vertices.

**Comment by Calinou** [comment]
This makes sense, but note that in the Compatibility rendering method, wireframes must be generated in advance. This is automatically done while in the editor in 3D (`RenderingServer.set_debug_generate_wireframes()`), so the same functionality would have to be added in 2D.

In the meantime, you could run Godot through [RenderDoc](https://renderdoc.org/) and get a wireframe view this way.
